### PR TITLE
Environments for integration tests

### PR DESCRIPTION
**Pull request by kerzhner**
https://github.com/statechannels/monorepo/pull/1501 precedes this PR.

Concentrate all environment variables for web3torrent integration tests in corresponding files. The benefits are:
- Easier to understand all environment variable values that are set for the tests.
- Makes it simpler to run the integration tests locally.
- Makes it simpler to play around with variables to try out different values.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -242,21 +242,9 @@ jobs:
...
       - image: circleci/node:10.16.3
     environment:
-      # For multiple
-      NODE_ENV: test
-      # For shared-ganache
-      USE_GANACHE_DEPLOYMENT_CACHE: true
-      GANACHE_CACHE_FOLDER: ../../.ganache-deployments
-      GANACHE_PORT: 8547
-      # For simple-hub
-      FIREBASE_PREFIX: << pipeline.git.branch >>-1
```
Comment (snario) [inline .circleci/config.yml:252]: I think this would need to be also set for `simple-hub`

**Comment by snario** [comment]
Also we could change the format of the files to `VAR_NAME = 'value'` like was set in #1508

File: .circleci/config.yml (hunk 1 above)
Comment (kerzhner) [reply .circleci/config.yml:252]: I dont think this integration test uses the hub.

**Comment by kerzhner** [comment]
> Also we could change the format of the files to `VAR_NAME = 'value'` like was set in #1508

Done in de0afa7a0

**Pull request merged**